letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -351,10 +351,22 @@ elif [ -f /etc/redhat-release ]; then
       BootstrapMessage "Legacy Red Hat-based OSes that will use Python3"
       BootstrapRpmPython3Legacy
     }
+    EnablePython36SCL() {
+      # Do nothing if Python 3.6 is already available
+      if ! python3.6 --version > /dev/null 2> /dev/null; then
+        # Do nothing if SCL Python 3.6 is not installed
+        if [ -f /opt/rh/rh-python36/enable ]; then
```

Comment:
So this is almost certainly fine, however, I think it'd be nice not to depend on this path never changing. There's a lot of code in the internals of `scl_source` allowing it to dynamically determine this path. What do you think about something like:
```
if scl --list 2>/dev/null | grep -q rh-python36; then
```

Reply:
Done.